src/account/accountSelectors.js

```diff
@@ -15,6 +15,7 @@ export const getAuth = (state: StateType) => {
       apiKey: undefined,
       email: undefined,
       realm: undefined,
+      gcmToken: ''
```

Comment:
This should not be part of the 'auth' structure.
We use it as a `something the api needs to authenticate` and nothing else will be added to it.